pages/wizard.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <v-container fill-height>
+    <v-row align="center" justify="center">
+      <v-col cols="12" sm="12" md="12" xl="8">
+        <h1 class="text-h4 mb-6 text-center">{{ heading }}</h1>
```

Comment:
Not sure if we should have a header here.

Technically, the stepper already has the title. By making the stepper vertical on small screen, we can keep the headers visible and not need this.